book/07-git-tools/sections/interactive-staging.asc

```diff
@@ -52,8 +52,8 @@ Update>> 1,2
 Update>>
 ----
 
-The `*` next to each file means the file is selected to be staged.
-If you press Enter after typing nothing at the `Update>>` prompt, Git takes anything selected and stages it for you:
+Ang `*` na susunod sa bawat file ay nangangahulugan sa file ay napili na i-stage.
+Kung ikaw ay pumindot sa Enter pagkatapos mag-type ng wala sa `Update>>` na prompt, ang Git kumukulang kung ano ang napili at i-stage ito para sa iyo:
```

Comment:
"nangangahulugan sa file ay napili" to "nangangahulugan na ang file ay napili"

Reply:
done change